src/universal/.devcontainer/devcontainer.json

```
@@ -43,8 +43,8 @@
             "additionalVersions": "3.1"
         },
         "ghcr.io/devcontainers/features/java:1": {
-            "version": "17",
-            "additionalVersions": "11",
+            "version": "21",
+            "additionalVersions": "11,17",
```

```suggestion
            "additionalVersions": "17",
```

For the universal image, we only support two latest LTS versions, so 21 and 17 makes sense. Can we make this change? Thanks!